### PR TITLE
Lei import

### PR DESCRIPTION
**Pull request by TomDemeranville**
Added lei importer.  Instructions in LoadLEIData.java comments:

The file can be found here: https://leidata.gleif.org/api/v1/concatenated-files/lei2/20180227/zip
 
1) wget file, 
2) unzip file
3) import:  `mvn exec:java -Dexec.mainClass=org.orcid.core.cli.LoadLEIData -Dorg.orcid.config.file=file:///Users/tom/git/ORCID-Source/orcid-persistence/src/main/resources/staging-persistence.properties -Dexec.args="-f path/to/extracted_file.xml"`
4) reindex: `mvn exec:java -Dexec.mainClass=org.orcid.core.cli.ProcessDisambiguatedOrgsForIndexing -Dorg.orcid.config.file=file:///path/to/ORCID-Source/orcid-persistence/src/main/resources/staging-persistence.properties`

Note there is a bug on the org typeahead.  You cannot look up things like 中航资本国际控股有限公司.  Not sure why. (you can look this entity up by its other name or LEI identifier)

### CODE REVIEW

**Comment by TomDemeranville** [comment]
[Add LEI](https://trello.com/c/D9vCcfJz/4591-add-lei)

File: orcid-solr-web/src/main/resources/log4j.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	<!-- Root Logger -->
 	<root>
 		<level value="DEBUG" />
-		<!-- <appender-ref ref="CONSOLE" /> -->
```
Comment (amontenegro) [inline orcid-solr-web/src/main/resources/log4j.xml]: Hey @TomDemeranville, shouldnt your rollback this?

**Comment by amontenegro** [comment]
Hey @TomDemeranville, could you please run the formater? it looks the java code doesn't follow our formatting rules.

Thanks

File: orcid-solr-web/src/main/resources/log4j.xml (hunk 1 above)
Comment (TomDemeranville) [reply orcid-solr-web/src/main/resources/log4j.xml]: Good catch!

**Comment by TomDemeranville** [comment]
OK, made the changes

**Pull request merged**